packages/storybook-vue/stories/components/beta-brand-header-navigation/BrandHeader.stories.mdx

```diff
@@ -97,9 +101,14 @@ export const Template = (args, { argTypes }) => ({
 - [`scale-telekom-mobile-menu-item`](https://github.com/telekom/scale/tree/main/packages/components/src/components/telekom/telekom-mobile-menu-item)
 
 ```html
-<scale-telekom-header>
-  <scale-telekom-nav-list slot="main-nav">
-    <scale-telekom-nav-item>
+<scale-telekom-header
+  meta-nav-aria-label="Meta navigation section" 
+  meta-nav-external-aria-label="External navigation section"
+  language-switcher-aria-label="Language switcher section"
+  main-nav-aria-label="Main navigation section"
```

Comment:
I wouldn't put this in the snippets, but rather only have them displayed in the props table… (you already [added them there](https://github.com/telekom/scale/pull/1738/files#diff-92d8def129d733c07189b404554219c9d2f9e16d2df3aa63a139a58d0517698a))

Reply:
I thought it would be useful to show since they are also added in the vue file, just to keep the code snippet consistent with the actual rendered html